custom-metrics-stackdriver-adapter/pkg/adapter/translator/response_translator_core.go

```diff
@@ -0,0 +1,101 @@
+package translator
+
+import (
+	"fmt"
+	"time"
+
+	stackdriver "google.golang.org/api/monitoring/v3"
+	apierr "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"sigs.k8s.io/metrics-server/pkg/api"
+)
+
+// GetCoreContainerMetricFromResponse for each pod extracts map from container name to value of metric and TimeInfo.
+func (t *Translator) GetCoreContainerMetricFromResponse(response *stackdriver.ListTimeSeriesResponse) (map[string]map[string]resource.Quantity, map[string]api.TimeInfo, error) {
```

Comment:
I assumed that you will include creation of PodResult in this refactor, not just code moving

Reply:
Done